### PR TITLE
GStreamer 1.12.3

### PR DESCRIPTION
**Pull request by philn**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/pygobject3.rb [hunk 1]
```diff
@@ -23,7 +24,9 @@ class Pygobject3 < Formula
 
   def install
     Language::Python.each_python(build) do |python, _version|
-      system python, *Language::Python.setup_install_args(prefix)
+      system "./configure", "--disable-dependency-tracking", "--prefix=#{prefix}", "PYTHON=#{python}"
+      system "make", "install"
+      system "make", "clean"
```
Comment (fxcoudert) [inline Formula/pygobject3.rb]: cc @tschoonj on that

**Comment by ilovezfs** [comment]
duplicates https://github.com/Homebrew/homebrew-core/pull/18260 but thanks for the PR nonetheless.

**Comment by philn** [comment]
Did you manage to build gst-python with current pygobject3?

**Comment by philn** [comment]
I can submit the pygobject3 commit in a separate PR. I needed that change to build gst-python here, FWIW :)

**Pull request closed without merging**